includes/modules/checkout_new_address.php

```diff
@@ -27,7 +27,7 @@
 
 if (isset($_POST['action']) && ($_POST['action'] == 'submit')) {
   // process a new address
-  if (zen_not_null($_POST['firstname']) && zen_not_null($_POST['lastname']) && zen_not_null($_POST['street_address'])) {
+  if (!empty($_POST['firstname']) && !empty($_POST['lastname']) && !empty($_POST['street_address'])) {

```

Comment:
May result in the same issue as with someone having the last name of 'Null' if they have the last name of 'false'.  Shouldn't an `isset` be incorporated instead of just `!empty`?

Reply:
`!empty` encompasses `isset`.  If the customer's last name is either 'Null' (a string) or 'false' (also a string), the result of !empty will be `(bool)true`.